### PR TITLE
Blog post: tus Node.js 1.0.0

### PR DESCRIPTION
**Pull request by Murderlon**

### CODE REVIEW

File: src/content/blog/2023-09-04-tus-node-server-v1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+revived and is now officially semver stable! It has been completely rewritten in
+TypeScript, split into separate packages, has new events and hooks system,
+elaborate docs with examples, and many fixes and features we’ll dive into below.
+
+## History
+
+The Node.js tus server, [tus-node-server][], was originally published by
+[@bhstahl][] in May 2016. As with many open source projects, not everyone has
+the time to maintain things for free in their own time and as such
+[tus-node-server][] was more or less abandoned at the end of 2018.
```
Comment (Acconut) [inline src/content/blog/2023-09-04-tus-node-server-v1.0.0.md:17]: I don't remember the exact details, but Ben Stahl didn't work on tus-node-server in his free time. At the time, he worked for Vimeo, so I assume that his involvement was work-related. He left Vimeo at the middle of 2018 and then tus-node-server was left abandoned: https://linkedin.com/in/bhstahl But I am not a 100% sure about this.

File: src/content/blog/2023-09-04-tus-node-server-v1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+In 2021, [@acconut][], [@mitjab][], and others sprung some live into the server
+again by upgrading dependencies adding some missing essential features. However
+there was still a lot of work ahead before the server could be considered stable
+and on par with the [protocol specification][].
+
+At the start of 2023, [Transloadit][] graciously allowed me ([@murderlon][]) to
+work on the server while getting paid. This, together with the help of
```
Comment (Acconut) [inline src/content/blog/2023-09-04-tus-node-server-v1.0.0.md:25]: Maybe we could link to https://transloadit.com/open-source/, where more details about Transloadit's open-source commitment are available.

File: src/content/blog/2023-09-04-tus-node-server-v1.0.0.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+      maxRetries: 10,
+    }),
+  },
+})
+const server = new Server({
+  path: '/files',
+  datastore,
+  // New hooks
+  async onIncomingRequest(req, res) {
+   // we can use this for access control
```
Comment (Acconut) [inline src/content/blog/2023-09-04-tus-node-server-v1.0.0.md:93]: Can we link to the documentation for hook here or somewhere else in the post?

**Comment by Acconut** [comment]
Overall a very informative post that sells tus-node-server quite well. If you are happy with the content, we can also ask AJ for his help.

File: src/content/blog/2023-09-04-tus-node-server-v1.0.0.md (hunk 1 above)
Comment (Murderlon) [reply src/content/blog/2023-09-04-tus-node-server-v1.0.0.md:17]: Perhaps it doesn't matter, the reason for abandoning it seemed to be life got in the way, how he worked on it during the maintenance period is not explicitly stated. Unless you feel it's important to attribute Vimeo?

Comment (kvz) [reply src/content/blog/2023-09-04-tus-node-server-v1.0.0.md:17]: I think it was Soundcloud actually where he worked when he did most of the tus-node-server work?

File: src/content/blog/2023-09-04-tus-node-server-v1.0.0.md (hunk 2 above)
Comment (kvz) [reply src/content/blog/2023-09-04-tus-node-server-v1.0.0.md:25]: >https://transloadit.com/open-source/

As folks with skin in the game, maybe also do a quick review on that page to see if it still makes sense and is written in the right tone of voice

File: src/content/blog/2023-09-04-tus-node-server-v1.0.0.md (hunk 1 above)
Comment (kvz) [reply src/content/blog/2023-09-04-tus-node-server-v1.0.0.md:17]: But yeah, maybe not relevant where he worked, employers never paid for him working on the project afaik anyway

**Review by Acconut — APPROVED** [review]
That's perfect 👌

**Pull request merged**